sdkv2.0.0/sdkv2.0.0/MojioSDK/MojioAuth.swift

```diff
@@ -49,8 +53,9 @@ class MojioAuth: NSObject, AuthControllerDelegate {
             let string = url.absoluteString;
```

Comment:
In Java rather than string parse URL parameters myself I use the common Uri class which handles a lot of edge cases (e.g. if the parameters appear in a different order and a different one is preceded with ? instead of &). It looks like you can use NSURLComponents to do this - I'd definitely recommend that approach instead.